docs/authentication/session-management.mdx

```diff
@@ -0,0 +1,896 @@
+---
+title: Session Management
+id: session-management
+description: ''
+slug: /session-management
+keywords:
+  - api token
+  - access token
+  - session management
+pagination_next: null
+pagination_prev: null
+last_update:
+  date: 08/08/2023
+  author: William May
+doc_type: overview
+displayed_sidebar: mainSidebar
+---
+
+import MultiLanguageCodeBlock from '@site/src/components/CodeBlocks/MultiLanguageCodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## What is Session Management
+
+Session management is a crucial aspect of web application development that involves securely managing user sessions. An authentication session is a period of user interaction with the application, typically starting with the user providing credentials and ending with logout or inactivity timeout. Effective session management ensures user data privacy, maintains session state, and provides a seamless user experience.
+
+The Beyond Identity session consists of something we call a `token`, specifically access tokens. An access token, also called an API token, is an object or file that enables access to an application or resource. The token is digitally signed and contains information about what resources can be accessed and for how long.
+
+## How to add session management to my application
+
+There are a few functions of tokens that you should be aware of. You can create a token, introspect a token, revoke a token, and retrieve active tokens.
+
+Applications may utilize cookies, local storage or other strategies to associate session with the end-user’s application. Beyond Identity doesn’t provide an API/SDK for this purpose; it is up to you to decide how to handle the end-user’s session. One option is store the JWT received from Beyond Identity in the end user’s application. Another solution is to store the JWT received from Beyond Identity on the server and use a cookie. This is a common solution offered by many web frameworks and is more secure as the JWT is more sensitive.
+
+### Create a Token
```

Comment:
I don't think we need the Create a token content in this article, as session mgt happens subsequent to requesting and obtaining a token. In my view we can just link to it and specify that once you obtain a token (access/refresh) then this article's guidance on session mgt details what to do with that token

Reply:
Based on the feedback, I think it might make more sense to remove the apis from this page and link to them instead, so this page can focus on the session management details.